lib/src/presentation/pages/home/home_screen.dart

```diff
@@ -50,37 +46,37 @@ class HomeScreen extends StatelessWidget {
                     return Column(
                       crossAxisAlignment: CrossAxisAlignment.start,
                       children: [
-                        CustomAppbarHome(),
-                        SizedBox(height: 16),
-                        LocationWidget(),
-                        SizedBox(height: 24),
+                        const CustomAppbarHome(),
+                        const SizedBox(height: 16),
```

Comment:
use vertical and horizontal instead of sized box and for height or width use h or w for responsive